### PR TITLE
fix websockets tracking tests

### PR DESCRIPTION
**Pull request by cezaraugusto**
Auditors: @diracdeltas
prep work for #11651

```
npm run test -- --grep="blocks websocket tracking"
```

### CODE REVIEW

File: test/bravery-components/braveryPanelTest.js [hunk 1]
```diff
@@ -453,38 +453,40 @@ describe('Bravery Panel', function () {
     it('blocks websocket tracking', function * () {
       const url = Brave.server.url('websockets.html')
       yield this.app.client
-        .waitForDataFile('adblock')
```
Comment (cezaraugusto) [inline test/bravery-components/braveryPanelTest.js]: this is already required in `beforeEach` hook so redundant

File: test/fixtures/websockets.html [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 <script>
   // This should get blocked by adblock if it is on
-  var exampleSocket = new WebSocket("ws://ag.innovid.com/dv/sync?tid=2");
+  var exampleSocket = new WebSocket("ws://wfpscripts.webspectator.com/bootstrap/ws-606FF556.js");
   exampleSocket.onerror = (e) => {
     document.getElementById('error').innerText = 'error'
   }
   // This should be a valid connection
-  var ws = new WebSocket("wss://echo.websocket.org")
+  var ws = new WebSocket("wss://websocketstest.com:443/service")
```
Comment (cezaraugusto) [inline test/fixtures/websockets.html]: uses same source as websocketstest.com script

File: test/lib/brave.js [hunk 3]
```diff
@@ -1073,7 +1073,7 @@ var exports = {
           logVerbose('waitForDataFile("' + dataFile + '") => ' + JSON.stringify(val.value[dataFile]))
           return val.value[dataFile] && val.value[dataFile].etag && val.value[dataFile].etag.length > 0
         })
-      }, 10000)
+      }, 30000)
```
Comment (cezaraugusto) [inline test/lib/brave.js]: getting data files from the store is very slow and often leads to intermittent so I added some more time. It's often not that long

Comment (luixxiul) [reply test/lib/brave.js]: Can this way be applied to https://github.com/brave/browser-laptop/issues/9059 ?

**Comment by diracdeltas** [comment]
the compact panel test failed: https://travis-ci.org/brave/browser-laptop/jobs/292458329#L4195. may be intermittent?

**Comment by cezaraugusto** [comment]
yes intermittent, it's passing locally. I'll try adding some more steps

File: test/lib/brave.js (hunk 3 above)
Comment (cezaraugusto) [reply test/lib/brave.js]: may fix too but lots of bravery panel tests are intermittent I'm trying to fix them all in #11651 which includes this too

**Comment by diracdeltas** [comment]
this test is failing https://travis-ci.org/brave/browser-laptop/jobs/292930564#L4394

**Comment by cezaraugusto** [comment]
this seems intermittent for a couple reasons. I'll close this and re-open when there's a window to investigate further, thanks for reviewing 👍

**Pull request closed without merging**